RELEASE_NOTES

```diff
@@ -0,0 +1,116 @@
+Release 0.9.0 (Draft)
+=====================
+
+Changes
+-------
+
+Changes that may break users' config files have been annotated with a `(!)`.
+
+- Note: mpv is not compatible with Lua 5.3. Lua 5.1 or 5.2 is required.
+- The minimum required libass version is now 0.12.1 or newer.
+- The minimum required FFmpeg version is now 2.4.0 (equiv. Libav 11) or newer.
+- `(!)` The internal libmpg123 support was removed.
```

Comment:
Maybe mention that it was unused by default in the previous release, even if it was compiled-in.